docs/ops/config.zh.md

```diff
@@ -104,7 +104,7 @@ The default fraction for managed memory can be adjusted using the taskmanager.me
 
 These parameters allow for advanced tuning. The default values are sufficient when running concurrent high-throughput jobs on a large cluster.
 
-{% include generated/netty_configuration.html %}
+{% include generated/network_netty_configuration.html %}
```

Comment:
It should also include this 

### Network Environment

{% include generated/network_environment_configuration.html %}

Reply:
This patch fixes the file path error, but I am fine to add you mention section to align with the config.